packages/twenty-emails/src/emails/clean-inactive-workspaces.email.tsx

```diff
@@ -1,9 +1,8 @@
-import * as React from 'react';
-import { BaseEmail } from 'src/components/BaseEmail';
-import { CallToAction } from 'src/components/CallToAction';
-import { HighlightedText } from 'src/components/HighlightedText';
-import { MainText } from 'src/components/MainText';
-import { Title } from 'src/components/Title';
+import { BaseEmail } from '../components/BaseEmail';
```

Comment:
@thaisguigon Not sure about using relative path import instead of absolute ones, as it's setup to be absolute in the front-end and back-end part
cc @charlesBochet

Reply:
I found a way to keep `twenty-emails`'s `baseUrl`, checking with @martmull to see if everything still works!